docs/paper/dev/getting-started/plugin-yml.md

```diff
@@ -0,0 +1,188 @@
+---
+slug: /dev/plugin-yml
+---
+
+# Paper Plugin YML
+
+The plugin.yml file is the main configuration file for your plugin. 
+It contains information about your plugin, such as its name, version, and description. 
+It also contains information about the plugin's dependencies, permissions, and commands.
+
+The plugin.yml file is located in the `resources` directory of your project.
+```
+src
+└── main
+    ├── java
+    └── resources
+        └── plugin.yml
```

Comment:
How about doing like this?
```plaintext
your_plugin_name
└──src
   └── main
       ├── java
       └── resources
           └── plugin.yml
```
This is trivial, though.

Reply:
Makes sense, have added